tests/integ/test_session.py

```diff
@@ -570,7 +570,7 @@ def test_use_secondary_roles(session):
     session.use_secondary_roles(current_role[1:-1])
 
 
-@pytest.mark.skipif(IS_IN_STORED_PROC, reason="SP doesn't allow to close a session.")
+@pytest.mark.skipif(IS_IN_STORED_PROC, reason="Can't create a session in SP")
```

Comment:
can this be reverted? old reason seems correct

Reply:
I can revert this.
I thought since now the closing of a session inside a stored procedure does not raise an error anymore it would make sense to adapt the message since the creation of a session is still raising an error.

Reply:
we can actually remove the skip since the test will no longer raise an error

Reply:
This is removed now 👍